### DISCUSSION TITLE
StreamingClient.filter not returning tweet entities

### DISCUSSION THREAD

**Discussion by pdeitel** [tweepy/tweepy#1967] [Questions] [answered]
I am filtering live tweets with a `StreamRule` containing `'football has:hashtags'`.

As expected, the returned tweets all contain `football` and include hashtags. 

When calling `filter`, I specifically passed `tweet_fields=['entities']` but the `StreamResponse` objects `includes` are all empty dictionaries. Any idea why that might be? Do I also need an expansion?

**Comment by Harmon758** [comment, marked as answer]
You're requesting the `entities` [Tweet field](https://tweepy.readthedocs.io/en/v4.10.1/expansions_and_fields.html#tweet-fields), so the Tweet objects in `StreamResponse.data` should include a populated [`Tweet.entities`](https://tweepy.readthedocs.io/en/v4.10.1/v2_models.html#tweepy.Tweet.entities) attribute.

`includes` is for objects requested through [expansions](https://tweepy.readthedocs.io/en/v4.10.1/expansions_and_fields.html#expansions).
